src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,31 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  todoCleaner: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, todoCleaner }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    setLoading(true);
+
+    getUser(todo.userId)
+      .then(setUser)
+      .then(() => setLoading(false));
```

Comment:
You are directly interacting with the DOM when setting the loading state. It's recommended to avoid direct DOM manipulations in React. Consider using a different approach, for example, by changing the component's state.